src/category_theory/monoidal/Mon_.lean

```diff
@@ -273,6 +297,158 @@ def equiv_lax_monoidal_functor_punit : lax_monoidal_functor (discrete punit.{u+1
 
 end Mon_
 
+namespace Mon_
```

Comment:
Could you add a `/-! ... -/` module doc here explaining what this section is building up to?